bin/kv_pair_appender.py

```diff
@@ -0,0 +1,32 @@
+# Key-value pairs appender I guess
+import os
+import json
+
+#going with relative paths
+script_path = os.path.dirname(os.path.abspath(__file__))
+dir_path = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "configs")
+
+# data that needs to be appended
+dd_app = {
+    "key_ob": {
+        "key1": "val1",
+        "key2": "val2"
+    }
+}
+
+# Iterate through the files in the configs dir
+for i in os.listdir(dir_path):
+    if i.endswith(".json"):
+        file_path = os.path.join(dir_path, i)
+
+        with open(file_path, "r") as file: 
+            try:
+                json_data = json.load(file)
+            except json.JSONDecodeError:
+                print(f"Error parsing JSON in file: {file_path}")
+                continue 
+
+        json_data.update(dd_app)
```

Comment:
you are going to run into an issue with this when you test because the newly added fields are not at the top level

Reply:
If I understand it correctly, it means that the `data_trajectories_columns_exclude ` should be under `admin_dashboard`
```
 "admin_dashboard": {
        "overview_users": true,
        "overview_active_users": true,
        "overview_trips": true,
        "overview_signup_trends": true,
        "overview_trips_trend": true,
        "data_uuids": true,
        "data_trips": true,
        "data_trips_columns_exclude": ["data.start_loc.coordinates", "data.end_loc.coordinates"],
        "additional_trip_columns": [],
        "data_uuids_columns_exclude": [],
        "token_generate": false,
        "token_prefix": "nrelop",
        "map_heatmap": true,
        "map_bubble": true,
        "map_trip_lines": false,
        "push_send": false,
        "options_uuids": true,
        "options_emails": true
    }
```